primitives/src/network.rs

```diff
@@ -0,0 +1,37 @@
+use codec::{Decode, Encode};
+#[cfg(feature = "std")]
+use serde::{Deserialize, Serialize};
+use sp_runtime::RuntimeDebug;
+
+/// Network type for parallel.
+#[derive(PartialEq, Eq, Clone, Copy, Encode, Decode, RuntimeDebug)]
+#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
+pub enum NetworkType {
+    Parallel,
+    Heiko,
+}
+
+impl NetworkType {
+    /// Return ss58 address prefix from network type.
+    pub fn ss58_addr_format_id(&self) -> u8 {
+        match self {
+            NetworkType::Heiko => 110,
+            NetworkType::Parallel => 172,
+        }
+    }
+
+    /// Return (token_symbol, token_decimal) of this network.
+    pub fn token_info(&self) -> (&str, u8) {
+        match self {
+            NetworkType::Heiko => (HEIKO_TOKEN, 12),
```

Comment:
The token decimals should not put in the chain_spec, not the primitives, so that won't miss use it. How about put the network source near to chain_spec (in same directory).

Reply:
Of course it's the better solution if we only use it in chain_spec, but maybe at least we also need it in other pallet's mock runtime for unit tests?

Reply:
The token decimals should never be used in runtime which lives on chain and hard to keep in sync with others chain.
For unit tests, hard code in test cases is fine to me.

Reply:
Got it. Fix it in next commit.

Reply:
@kaichaosun Every token has registered the decimal when applying the ss58 prefix. Can we use the token decimals as same as the declared in the substrate s58-registry.json? 
https://github.com/paritytech/substrate/pull/9254/files#diff-b501fa660c67a198b63b9d0ed792673bb94d91c275dfcf2d250ce06eafb66419R540

Reply:
I'm not sure, but a trivial solution is fetch and parse it in build script.

Reply:
I'm not keen to use the json file, as it's prepared for frontend if i remember correctly.